docs/src/IceNucleationParcel0D.md

```diff
@@ -353,46 +353,47 @@ Three different implementations of this parametrization are used in the parcel m
   with the mean and standard deviation defined in [Frostenberg2023](@cite).
   The inverse timescale of the process is set by ``\gamma``.
 
-The stochastic implementation is based on the equation for a generic
-  stochastic process ``x`` with Gaussian random noise (i.e. Wiener process):
+The stochastic implementation is based on the [Ornstein-Uhlenbeck process](https://en.wikipedia.org/wiki/Ornstein–Uhlenbeck_process), 
+in which the variable ``x`` is a mean-reverting process perturbed by Gaussian random noise (i.e. Wiener process):
 ```math
 \begin{equation}
-  dx = - \gamma \; x \; dt + g dW; \;\;\;\;\;\;\;   dW = {\bf{N}}(0, dt^2),
+  dx = - \gamma(x - \mu)dt + \sqrt{2\gamma} \sigma dW; \quad\quad   dW \sim \mathcal{N}(0, dt),
```

Comment:
`mathcal` doesn't render properly. I think it's ok to stick with `bf`

Reply:
Also, what is the justification for multiplying the second term by `\sqrt{2\gamma}` ? Thats not how the problem is formulated in the wiki link you provided?